### PR TITLE
Fix neovim release link

### PR DESCRIPTION
**Pull request by baptisteArno**

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ If you are on AstroNvim v2.0 and are upgrading to AstroNvim v3.0 please see the
 ## ⚡ Requirements
 
 - [Nerd Fonts](https://www.nerdfonts.com/font-downloads) (_Optional with manual intervention:_ See [Recipes/Customizing Icons](Recipes/icons#disable-icons))
-- [Neovim 0.9 (_Not_ including nightly)](https://github.com/neovim/neovim/releases/tag/v0.9.0)
+- [Neovim (_Not_ including nightly)](https://github.com/neovim/neovim/releases/tag/stable)
```
Comment (mehalter) [inline docs/intro.md:23]:
```suggestion
- [Neovim 0.8+ (_Not_ including nightly)](https://github.com/neovim/neovim/releases/tag/stable)
```

Could you please revise this to say

**Review by mehalter — CHANGES_REQUESTED** [review]
Thanks for catching this broken link! Could you please just fix the one suggestion to the link text?

**Review by mehalter — APPROVED** [review]
I made a small revision and this looks great thanks for catching this broken link!

**Comment by baptisteArno** [comment]
Hey thanks, sorry was not available yesterday to apply the changes 🙏

**Pull request merged**